### ISSUE TITLE
it can't upgrade to the latest version

### ISSUE THREAD

**Issue by TheoShira** [UndertakerBen/PorOperaUpd#4]
it can't upgrade to the latest version now, Updater always stays at version 86.0.4363.59, But, Up to now, the latest version should be 89.0.4447.51
![图片](https://user-images.githubusercontent.com/39277263/181217399-08cf028e-9921-4123-8d44-0471f9662edb.png)

**Comment by UndertakerBen** [comment]
Thanks for reporting this issue. It's fixed now with version 1.3.2.0 of the updater.

**Comment by TheoShira** [comment]
Thanks，And could you please make a launcher for Whale?
url: https://whale.naver.com/en/download/win/
It is a good browser to use

**Comment by TheoShira** [comment]
At day 2023/2/24，it can't upgrade to the latest version again, Updater always stays at version 95.0.4635.46, But, Up to now, the latest version should be 96.0.4693.20

**Comment by hogiro** [comment]
> At day 2023/2/24，it can't upgrade to the latest version again, Updater always stays at version 95.0.4635.46, But, Up to now, the latest version should be 96.0.4693.20 ![image](https://user-images.githubusercontent.com/39277263/221002695-86fc9884-135e-4504-a0d8-d3e6bf1b5112.png)

yes right. It is because the URL, from which he is getting the update information, is showing only these versions. He needs to update the URL to get the right information.
